modules/microshift-ingress-controller-config.adoc

```diff
@@ -91,21 +116,118 @@ ingress:
 |Parameter |Description
 
 |`ingress`
-|The `ingress` section of the {microshift-short} `config.yaml` file defines the configurable parameters for the implemented portions of the {OCP} Ingress Control Operator. All parameters in the rest of this table are subsections in the `ingress` section of the `config.yaml`.
+|The `ingress` section of the {microshift-short} `config.yaml` file defines the configurable parameters for the implementation of the {OCP} `IngressController` API. All of the following parameters in this table are subsections in the `ingress` section of the {microshift-short} `config.yaml`.
+
+|`accessLogging`
+|This `ingress` subsection describes how client requests are logged. If the `status` field is empty, access logging is disabled. When the status field is set to `Enabled`, access requests are logged as configured with the `accessLogging` parameters and the `accessLogging.destination.type` is automatically set to `Container`.
+
+* When enabled, access logging is part of the `openshift-router` logs. The sos report procedure for {microshift-short} captures logs from this pod.
+
+|`accessLogging.destination`
+|A destination for logs. The destination for logs can be a local sidecar container or remote. Default value is null.
+
+|`accessLogging.destination.type`
+|The type of destination for logs. Valid values are `Container` or `Syslog`.
+
+* Setting this value to `Container` specifies that logs should go to a sidecar container. When the destination type is set to `Container`, a container called `access-log` is automatically created. Using container logs means that logs might be dropped if the rate of logs exceeds the container runtime capacity or the custom logging solution capacity. You must have a custom logging solution that reads logs from this sidecar.
```

Comment:
Sorry, the container should be `access-logs`, not `access-log`. Others LGTM, thanks.